pkgs/by-name/ne/neohtop/package.nix

```
@@ -0,0 +1,78 @@
+{
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  jq,
+  stdenv,
+  fetchNpmDeps,
+  zsh,
+  fetchFromGitHub,
+  npmHooks,
+  lib,
+  openssl,
+}:
+
+let
+  tauri = cargo-tauri;
+
+in
+rustPlatform.buildRustPackage rec {
+  name = "neohtop";
+  pname = name;
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "abdenasser";
+    repo = "neohtop";
+    rev = "v${version}";
```

Prefix tags with refs/tags to avoid collision with branches
```suggestion
    rev = "refs/tags/v${version}";
```